libraries/AP_Logger/LoggerMessageWriter.cpp

```diff
@@ -37,10 +37,6 @@ void LoggerMessageWriter_DFLogStart::process()
 {
     switch(stage) {
     case ls_blockwriter_stage_init:
-        stage = ls_blockwriter_stage_formats;
-        FALLTHROUGH;
-
-    case ls_blockwriter_stage_formats:
```

Comment:
init is the stage which could go away here (if any!), not _formats.  It's just a name thing...